plugins/http_plugin/http_plugin.cpp

```diff
@@ -195,9 +197,16 @@ namespace eosio {
          template<class T>
          void handle_http_request(typename websocketpp::server<detail::asio_with_stub_log<T>>::connection_ptr con) {
             try {
+               bool is_secure = con->get_uri()->get_secure();
+               const auto& local_endpoint = con->get_socket().lowest_layer().local_endpoint();
+               auto local_socket_host = local_endpoint.address().to_string();
+               auto port = local_endpoint.port();
+               if (!(is_secure && port == 443) || (!is_secure && port == 80)) {
```

Comment:
Is there a reason to restrict this to 443 for https and 80 for http? Seems like you could enable no matter what local_endpoint port is used.